rasa_nlu/data_router.py

```diff
@@ -336,6 +343,28 @@ def training_errback(failure):
         self._current_training_processes += 1
         self.project_store[project].current_training_processes += 1
 
+        # tensorflow training is not executed in a separate thread, as this may
+        # cause training to freeze
+        if self._tf_in_pipeline(train_config):
+            try:
+                model_path = do_train_in_worker(
```

Comment:
Please add a logger warm her before the call to inform the user that this will block the server